src/style.css

```
@@ -1 +1,20 @@
 /* styles go here */
+.mb10 {
+  margin-bottom: 10px;
+}
+
+.formLabel {
+  display: block;
+}
+
+.formLabel:last-child {
+  margin-bottom: 0;
+}
+
+.wrapper {
```

Краще задати конкретнішу назву для селектора, бо обгортка звучить дуже абстрактно
```suggestion
.fieldset {
```